src/components/views/Registration.tsx

```diff
@@ -155,12 +106,8 @@ const Registration = () => {
                   Register
                 </Button>
               </div>
+              <ErrorMessages errors={errors} generalError={generalError} />
             </div>
-            {getAllErrorMessages().map((error, index) => (
-              <div key={index} className="error-message">
-                {error}
-              </div>
-            ))}
             <label className="register message">
```

Comment:
please put the error message outside of the container like Login.tsx and add a margin for consistent styling